config/tasklists/learn-to-drive-a-car.json

```diff
@@ -0,0 +1,154 @@
+{
+  "title": "Learn to drive a car: step by step",
+  "base_path": "/learn-to-drive-a-car",
+  "description": "Check what you need to do to learn to drive.",
+  "links": {
+    "breadcrumbs": [
+      { "title": "Home", "url": "/" },
+      { "title": "Driving and transport", "url": "/browse/driving"},
+      { "title": "Learning to drive", "url": "/browse/driving/learning-to-drive"}
+    ]
+  },
+  "tasklist": {
+    "small": true,
+    "heading_level": 3,
+    "steps": [
+      [
+        {
+          "title": "Check you're allowed to drive",
+          "panel_descriptions": ["Most people can start learning to drive when they’re 17."],
+          "panel_links": [
+            {
+              "href": "/vehicles-can-drive",
+              "text": "Check what age you can drive"
+            },
+            {
+              "href": "/legal-obligations-drivers-riders",
+              "text": "Requirements for driving legally"
+            }
```

Comment:
We now also have driving eyesight rules here, see https://docs.google.com/spreadsheets/d/1A6pWxHloiMvExtiQNmL-8m-ruXQFRluNObK28uhU1jI/edit#gid=1049909139

Reply:
Did you change the json file? I've copied it from the collections app at the time

Reply:
Yep, it's up to date in frontend and collections.  The collections one doesn't have `small` or the heading level so careful when copying it :-D